### ISSUE TITLE
LLVM binary installer 

### ISSUE THREAD

**Issue by ghost** [WebAssembly/binaryen#1200]
Will Web Assembly eventually get an installable binary for Emscripten? I mean for each platform or starters will continue to build from source

**Comment by kripken** [comment]
The emsdk should download binaries for all platforms now, I think. Make sure you are using the latest version of the emsdk. cc @juj

**Comment by QuantiumK** [comment]
I just want to get started with web assembly. The docs at
http://webassembly.org said I have to build from source.
I just want to know if that will change soon.

**Comment by kripken** [comment]
Yeah, those docs look out of date. The main emsdk docs are [here](https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html), and they mention that you don't need to build from source. I verified now on linux, after downloading the emsdk from that link,
````
./emsdk update
./emsdk install latest
./emsdk activate latest
````
That downloaded and set up everything, without needing to build from source.

I also opened a PR to update those docs, https://github.com/WebAssembly/website/pull/107
